### PR TITLE
Update thank you page skeleton styles

### LINKED ISSUE #74292: Marktplace: Update thank you page skeleton
This task will focus on the main changes designed in the figma link 5ddl1WFTEnaIatL2WlZp4k-fi-2459%3A3382

### PR DESCRIPTION
**Pull request by gcsecsey**
Closes #74292

## Proposed Changes

* Update the styling of the thank you page skeleton

## Testing Instructions

* Naviagate to the thank you page
* Compare the styling of the bottom sections to the Figma designs
* Check for any regressions

| Before | After |
| ------ | ----- |
| image | image | 

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/checkout/checkout-thank-you/marketplace/style.scss [hunk 1]
```diff
@@ -87,16 +85,21 @@
 				line-height: 20px;
 				font-size: $font-body-small;
 				color: var(--studio-gray-60);
+				margin-bottom: 16px;
```
Comment (gcsecsey) [inline client/my-sites/checkout/checkout-thank-you/marketplace/style.scss:88]: This is needed to override the default styling in https://github.com/Automattic/wp-calypso/blob/trunk/client/components/thank-you/style.scss#L50

File: client/my-sites/checkout/checkout-thank-you/marketplace/style.scss [hunk 2]
```diff
@@ -87,16 +85,21 @@
...
 		.thank-you__step-cta {
 			margin: 0;
-		}
 
-		.thank-you__step-icon {
-			width: calc(18px + 23px); // icon size + padding
-			flex-basis: 100%;
+			a {
+				font-weight: 500;
+				font-size: $font-body;
```
Comment (epeicher) [inline client/my-sites/checkout/checkout-thank-you/marketplace/style.scss]: According to the design, I think this should be
```suggestion
				font-size: $font-body-small;
```

**Review by epeicher — COMMENTED** [review]
I have checked it, and it looks fine. I would reduce a bit the gap between the footer columns [here](https://github.com/Automattic/wp-calypso/pull/74575/files#diff-84af283878bb5d44f451392293e82a3fc8c07776d86c8fb60f7df6f373c42823R58) (GitHub does not let me add a comment there) to `40px` 
CleanShot 2023-03-20 at 10 52 09@2x
And I have left a comment regarding the font size of the link, but overall this looks great

File: client/my-sites/checkout/checkout-thank-you/marketplace/style.scss (hunk 2 above)
Comment (gcsecsey) [reply client/my-sites/checkout/checkout-thank-you/marketplace/style.scss]: Great points @epeicher, thank you for the thorough review! 🙏

**Review by epeicher — APPROVED** [review]
Thanks for addressing the comments 🙏 . I have checked the styles, and they LGTM! 🚢

**Pull request merged**